models/issue.go

```diff
@@ -180,6 +180,9 @@ func (issue *Issue) loadPullRequest(e Engine) (err error) {
 		}
 		issue.PullRequest.Issue = issue
 	}
+	if issue.PullRequest != nil && issue.PullRequest.Issue == nil {
```

Comment:
Any reason to not simplify it like this?

```suggestion
	if issue.PullRequest != nil {
```

Reply:
I just didn't want to change the reference if it's already set.

Reply:
@oscarcosta why not use `.LoadIssue()` it does exact this

Reply:
I would not add this in here! (issue.PR.issue = issue) sounds like a bad idear!
just pull.LoadIssue() when you need it!

Reply:
this was not exactly what I ment ... I'll send a pull-request ...